music-player/src/components/PlaylistsComponent.vue

```diff
@@ -48,6 +48,56 @@
       </q-badge>
       <q-badge color="red" @click="deletePlaylist(index)"> DELETE </q-badge>
     </div>
+
+    <q-btn
```

Comment:
It should be declared in SongsCompinent from the way I see it! My understanding was that We're gonne add new songs on playlist perspective. To discuss